.github/workflows/cargo.yml

```diff
@@ -32,7 +32,9 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
     - name: Prepare Machine
-      run: scripts/prepare-machine.ps1 -Tls quictls -ForBuild -InstallTestCertificates
+      run: |
+        scripts/prepare-machine.ps1 -Tls quictls -ForBuild -InstallTestCertificates
+        git submodule update --init
```

Comment:
Please revert this. `prepare-machine.ps1` should be doing any submodule init as necessary.